<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createFileDialog</class>
 <widget class="QDialog" name="createFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,60,10,20">
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="m_labelDetailed">
     <property name="text">
      <string>Create a new source file and add it to your project.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="40,20,40">
     <property name="spacing">
      <number>30</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="30,70">
       <item>
        <widget class="QLabel" name="m_labelFileType">
         <property name="text">
          <string>File type:</string>
         </property>
         <property name="buddy">
          <cstring>m_comboxFileType</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_comboxFileType">
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtCurrent</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="30,70">
       <item>
        <widget class="QLabel" name="m_labelFileName">
         <property name="text">
          <string>File name:</string>
         </property>
         <property name="buddy">
          <cstring>m_lineEditFileName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lineEditFileName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="30,70">
       <item>
        <widget class="QLabel" name="m_labelFileLocation">
         <property name="text">
          <string>File location:</string>
         </property>
         <property name="buddy">
          <cstring>m_comboxFileLocation</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_comboxFileLocation">
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtCurrent</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushBtnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushBtnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
